### PR TITLE
migrate from vanillajs to react

### PR DESCRIPTION
**Pull request by Atakannbal**
#4 

Following up on this migration

@admkc

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import React, { useState, useEffect } from 'react'
+const { ipcRenderer } = require('electron')
+
+const getFilePathFromEvent = (event) => {
+  return event.dataTransfer.files[0].path;
+}
+
+const getSubFolders = (file) => {
```
Comment (ademkoc) [inline src/App.js]: We can change the parameter name. if we rename it to _folder_, it can be more nice

**Review by ademkoc — CHANGES_REQUESTED** [review]
All is fine, just we need to rename one parameter

File: src/App.js (hunk 1 above)
Comment (Atakannbal) [reply src/App.js]: I've missed it, you are right. Thanks!

**Review by ademkoc — APPROVED** [review]
+1

**Pull request merged**